File: web/src/Model/DevisSpectacle.ts
import {DepenseArtistes, Depensedivers, DepenseElements, DepenseLieu, Devis} from "./Devis";

export class DevisSpectacle{
    devis:Devis=new Devis()
    depenseLieus:DepenseLieu=new DepenseLieu()
    artistes:DepenseArtistes=new DepenseArtistes()
    depenseElements:DepenseElements=new DepenseElements()
    depensedivers:Depensedivers=new Depensedivers()
}

File: ServerApp/src/main/java/com/example/serverapp/Model/ViewModel/DevisSpectacle.java
package com.example.serverapp.Model.ViewModel;

import com.example.serverapp.Model.MainModel.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class DevisSpectacle {
    Devis devis;

    List<DepenseLieu> depenseLieus;

    List<DepenseArtistes> artistes;

    List<DepenseElements> depenseElements;

    List<Depensedivers> depensedivers;

}
